src/styles/components/messages.css

```
@@ -152,80 +164,82 @@
+}
+
+#messages .message-bubble .message-text {
+  color: var(--text-primary);
+
+  padding: 2px 6px;
+
+  font-size: 14px;
+  line-height: 1.2rem;
+  letter-spacing: 0.3px;
+
+  white-space: pre-wrap; /* Preserve line breaks from Shift+Enter */
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)